service/src/test/java/uk/gov/hmcts/reform/fpl/service/email/content/OrderIssuedEmailContentProviderTest.java

```diff
@@ -55,6 +56,9 @@
     @MockBean
     private OrderCreationService orderCreationService;
 
+    @MockBean
+    private ManageOrdersClosedCaseFieldGenerator manageOrdersClosedCaseFieldGenerator;
```

Comment:
If you want, you can this to the list of classes in line 43 (since you're not mocking this).